lib/installation.js

```diff
@@ -0,0 +1,33 @@
+var _ = require('lodash');
+
+var model = require('./model');
+var pagination = require('./pagination');
+
+var Installation = model.create(function() {});
+
+// Create a request url
+Installation.prototype.url = function() {
+    return _.compact(
+        ['/installation'].concat(_.toArray(arguments))
+    ).join('/');
+};
+
+// List installation repositories
+Installation.prototype.repos = pagination(function(userId) {
+    var params = {};
+    if (userId) {
+        params.user_id = userId;
+    }
+
+    return {
+        url: this.url('repositories'),
+        query: params,
+        opts: {
+            headers: {
```

Comment:
Move this as constant for the whole file `lib/installation.js` since other methods will require the same headers.